wallet/src/de/schildbach/wallet/ui/staking/StakingActivity.kt

```diff
@@ -96,6 +97,13 @@ class StakingActivity : LockScreenActivity() {
         when (status) {
             OnlineAccountStatus.None -> { }
             OnlineAccountStatus.Linking, OnlineAccountStatus.SigningUp -> super.turnOffAutoLogout()
+            OnlineAccountStatus.Validating -> {
+                val isWebView = navController.currentDestination?.id == R.id.crowdNodeWebViewFragment
+                if (isWebView) {
+                    navController.navigate(WebViewFragmentDirections.webViewToPortal())
+                }
+                super.turnOnAutoLogout()
+            }
```

Comment:
@Syn-McJ This is your suggestion and it works for me.

Reply:
Yes, this should work